### ISSUE TITLE
Strapi 

### ISSUE THREAD

**Issue by chris-smearing** [strapi/strapi-examples#17]
**Node.js version**:
v9.5.0

**npm version**:
5.6.0

**Strapi version**:
3.0.0-alpha.11.1

**Gatsby version**:
1.1.45

**Operating system**:
Windows 10

**Which example is causing problem?**
Gatsby blog

**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
error on final step of the tutorial: https://hackernoon.com/building-a-static-blog-using-gatsby-and-strapi-8b5acfc82ad8

After adding the function to fetch authors, the following is printed to the console during the compilation:

> The new node didn't pass validation
> ValidationError: child "id" fails because ["id" is required]

```javascript
{ username: 'chris',
  ...
  id: undefined,
  ...
}
```

**If the current behavior is a bug, please provide the steps to reproduce the problem**
Follow the tutorial, but use a newer version of Strapi

**What is the expected behavior?**
id should probably have a value.

I don't know either framework very well, or I might have tried modifying the plugin to get the id field from _id.

**Comment by chence** [comment]
same error +1

**Comment by robhowell** [comment]
me too +1

**Comment by nom4da** [comment]
same here +1

**Comment by imjustd** [comment]
same +1

**Comment by ymek** [comment]
While this clearly isn't a code fix, the following will get people going:

Connect to MongoDB's mongo-shell:
```shell
mongo
```

After you're connected, insert the missing data.
```shell
use strapi
var myObjectId = db['users-permissions_user'].findOne()._id
db['users-permissions_user'].findOneAndUpdate({ _id: { $eq: myObjectId } }, { $set: { id: myObjectId.valueOf() } })
```

Interestingly enough, this specific issue is not present when creating a user in the Strapi control panel.

**Comment by pierreburgy** [comment]
Good catch! I submitted a PR for the `gatsby-source-strapi` plugin: https://github.com/strapi/gatsby-source-strapi/pull/4. 

As soon as it is published on GitHub, I will update this project.

**Comment by entrptaher** [comment]
@pierreburgy everything is nice and dandy, except this isn't updated yet and I just wasted an hour trying to understand why I was getting error when I used the example repo. I thought it was updated even on the example repo. :'( 

https://github.com/strapi/strapi-examples/blob/master/gatsby-strapi-tutorial/blog/package.json#L10
